src/java.base/share/classes/jdk/internal/loader/BuiltinClassLoader.java

```diff
@@ -355,38 +355,38 @@ public URL findResource(String name) {
      */
     @Override
     public Enumeration<URL> findResources(String name) throws IOException {
-        List<URL> checked = new ArrayList<>();  // list of checked URLs
+        List<URL> resources = new ArrayList<>();  // list of resource URLs
 
         String pn = Resources.toPackageName(name);
         LoadedModule module = packageToModule.get(pn);
         if (module != null) {
 
             // resource is in a package of a module defined to this loader
             if (module.loader() == this) {
-                URL url = findResource(module.name(), name); // checks URL
+                URL url = findResource(module.name(), name);
                 if (url != null
                     && (name.endsWith(".class")
                         || url.toString().endsWith("/")
                         || isOpen(module.mref(), pn))) {
-                    checked.add(url);
+                    resources.add(url);
                 }
             }
 
         } else {
             // not in a package of a module defined to this loader
             for (URL url : findMiscResource(name)) {
                 if (url != null) {
-                    checked.add(url);
+                    resources.add(url);
                 }
             }
         }
 
-        // class path (not checked)
+        // class path
         Enumeration<URL> e = findResourcesOnClassPath(name);
 
-        // concat the checked URLs and the (not checked) class path
+        // concat the resources from the modules and the class path
```

Comment:
Observation: This is now essentially `ClassLoader.CompoundEnumeration`, which could be reused if moved to `jdk.internal.loader`. Probably not for this PR though.

Reply:
The new comment isn't quite right, it should say "concat the URLs to the resource in the modules and the class path".

Reply:
> concat the URLs to the resource in the modules and the class path

Did you mean "concat the URLs of the ...." ?

Reply:
"of" or "to" will work here.

Reply:
Done - I've now updated this comment.